app/views/devise/sessions/new.html.slim

```diff
@@ -1,25 +1,28 @@
-h2
-  | Log in
-
-= render 'shared/notifications'
```

Comment:
@MaximilianMeister the issue is about implementing fully styled errors. If that is going to take a long time we can go with an intermediate fix for now. In any case, I agree, lets solve this in another PR.